styles/style.scss

```diff
@@ -191,10 +195,140 @@ label {
   font-size: 0.8rem;
 }
 
+.modal {
+  display: none;
+  position: absolute;
+  z-index: 9999;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  min-width: 80%;
+  height: auto;
+  background-color: white;
+  border-radius: 16px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+  padding: 2rem 2.5rem;
+
+  &__title {
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: 600;
+    font-size: 1.4rem;
+    line-height: 1.7rem;
+  }
+
+  &__description {
+    padding-top: 0.5rem;
+    padding-bottom: 1.67rem;
+  }
+
+  &__answers {
+    display: flex;
+
+    &--img {
+      height: 11.5rem;
+    }
+  }
+
+  .table {
+    margin-left: 1.83rem;
+    text-align: left;
+
+    caption {
+      text-align: left;
+      font-size: 0.87rem;
+      line-height: 1.07rem;
+      font-weight: 600;
+      padding-bottom: 0.6rem;
+    }
+
+    th {
+      color: rgba(0, 0, 0, 0.6);
+    }
+
+    th,
+    td {
+      font-size: 0.8rem;
+    }
+
+    td {
+      padding-right: 1.83rem;
+    }
+
+    .table__cell {
+      &--correct {
+        color: rgba(87, 181, 13, 0.6);
+      }
+
+      &--incorrect {
+        color: rgba(255, 0, 0, 0.6);
+      }
+    }
+
+    &__img {
+      height: 2.87rem;
+      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+        4px 4px 40px rgba(255, 0, 0, 0.9);
+      border-radius: 16px;
+      transform: matrix(-1, 0, 0, 1, 0, 0);
+    }
+  }
+
+  .form {
+    display: flex;
+
+    &__input {
+      border: 4px solid #c4c4c4;
+      border-radius: 8px;
+      height: 2rem;
+      width: 16rem;
+      font-size: 1.2rem;
+      padding: 1rem 1rem;
```

Comment:
jak dajesz obie wartosci na 1rem to mozna dac to po prostu raz :)

Reply:
hehe